cognite/neat/rules/exporter/rules2ontology.py

```diff
@@ -250,31 +261,30 @@ def from_list_of_properties(cls, definitions: list[Property], namespace: Namespa
 
         if not cls.same_property_id(definitions):
             raise exceptions.PropertyDefinitionsNotForSameProperty()
-
-        prop_dict = {
-            "id_": namespace[definitions[0].property_id],
-            "type_": set(),
-            "label": set(),
-            "comment": set(),
-            "domain": set(),
-            "range_": set(),
-        }
-
+        instance = cls.model_construct(
```

Comment:
Rename `instance` to `owl_property` or `owl_property_definition`, since `instance` has different meaning here (even though you meant instance of given class)